project.clj

```diff
@@ -8,7 +8,7 @@
   :pedantic? :abort
   :dependencies [[org.clojure/clojure]
                  [org.clojure/java.jdbc]
-                 [org.clojure/test.check "0.9.0"]
+                 [org.clojure/test.check "1.1.1"]
```

Comment:
Shouldn't this be in the test dependencies so it's not transitively picked up by other projects?

Then this whole situation goes away.

Reply:
Probably, come to think of it. A thing that may not have been known when it was added 8 years ago 🤔 Happy to move it there if we think that's fine.

Reply:
It does only seem to be used in tests (wanted to make sure it wasn't used in helpers for other libraries' tests). I'll move it.